scipy/optimize/_numdiff.py

```diff
@@ -240,17 +244,29 @@ def approx_derivative(fun, x0, method='3-point', rel_step=None, f0=None,
 
         Note, that sparse differencing makes sense only for large Jacobian
         matrices where each row contains few non-zero elements.
+    as_linear_operator : bool, optional
+        When `as_linear_operator` is True the function returns an
+        `LinearOperator`. Otherwise it returns an dense array or sparse
```

Comment:
Ok! I will do it in the place you have indicated and keep the short notation `LinearOperator` everywhere else.